examples/simple-platform/configure.adoc

```diff
@@ -0,0 +1,251 @@
+application screen on GitHub", 1280, 623, role="frame"]
+
+. In the **Application name** field, enter a name of your choice. For
+  example, `Interbit Test App`.
+
+. In the **Homepage URL** field, enter a URL. Any URL works since we are
+  running the example locally.
+
+. In the **Authorization callback URL** field, enter
+  `http://localhost:8888/oauth/github`.
++
+NOTE: The callback URL is case sensitive, and the port specified is for
+running the `web-auth-endpoint` node locally.
+
+. image:img/button-github-register_app.png["The Register app button on
+  GitHub", 163, 34, role="right"]
+  Click the **Register application** button to finish creating the OAuth
+  app.
++
+IMPORTANT: Leave this browser tab open as we are going to copy values
+from this tab in subsequent steps.
+
+
+== Clone the `interbit` repository and install `interbit-cli`
+
+. Open a terminal and clone the `interbit` repository from GitHub:
++
+[source,bash]
+git clone https://github.com/interbit/interbit.git
+
+. Install the link:/reference/interbit-cli/README.md[`interbit-cli`]
+  package globally:
++
+[source,bash]
+npm i -g interbit-cli
+
+
+== Generate a set of private and public keys for the `platform-deploy` and `web-auth-endpoint` hypervisors
+
+. Create a `secrets` directory at the root of the `interbit` repository.
++
+The `secrets` directory is included in this repository's `.gitignore`
+file and is not tracked by git. We don't want to commit the files we are
+going to add here because they include private keys for our Interbit
+nodes.
+
+. Change the current working directory to the `secrets` directory:
++
+[source,bash]
+cd secrets
+
+. Generate Interbit key pairs
++
+Both the `platform-deploy` and `web-auth-endpoint` apps require public
+and private keys. The following commands generate them and store them in
+JSON files:
++
+[source,bash]
+interbit keys --filename platform-deploy-keys.json
+interbit keys --filename web-auth-endpoint-keys.json
+
+
+== Configure environment variables for `platform-deploy`
+
+. Create the file `platform-deploy.sh` in the `secrets` directory with
+  the following content:
++
+[source,bash]
+----
+#!/bin/bash
+# Secrets for Accounts app GitHub OAuth chain
+export GITHUB_CLIENT_ID=""
+export GITHUB_CLIENT_SECRET=""
+export GITHUB_REDIRECT_URL=""
+export OAUTH_CALLBACK_URL="http://localhost:3025/account/oauth/gitHub"
+# Key pair for the platform node
+export PUBLIC_KEY=""
+export PRIVATE_KEY=""
+# Peer list override
+export PORT=5025
+export CONNECT_TO_PEERS="localhost:8888"
+----
+
+. Update the configuration in `platform-deploy.sh`:
++
+--
+[loweralpha]
+. Copy the `Client ID` value from the GitHub browser tab and paste it
+  into the `GITHUB_CLIENT_ID` definition in `platform-deploy.sh`.
+
+. Copy the `Client Secret` value from the GitHub browser tab and paste
+  it into the `GITHUB_CLIENT_SECRET` definition in `platform-deploy.sh`.
+
+. In `Authorization callback URL` definition in `platform-deploy.sh`,
```

Comment:
It's the `GITHUB_REDIRECT_URL` definition that gets the specified value. This value should be the same as the `Authorization callback URL` specified when you created the GitHub OAuth app.